src/test/java/com/ncc/aif/NistExamplesAndValidationTest.java

```diff
@@ -311,11 +333,17 @@ void valid() {
             void invalid() {
                 // Test entity, relation, and event. Correct other than being clustered
                 final Resource newEntity = makeEntity(model, utils.getEntityUri(), system);
-                markJustification(utils.addType(newEntity, SeedlingOntology.Weapon), utils.makeValidJustification());
+                markJustification(utils.addType(newEntity,
+                        LDCOntology.WEA),
+                        utils.makeValidJustification());
```

Comment:
There really are a lot of these, and to add insult to injury, it's breaking it up in the middle of the `addType()` call.  Seems like it could be just be on one line like before.

Reply:
I know.  I went back and forth before I decided that consistent style of putting these constants on their own line was preferable.  We may decide to make this a more specific weapon, which would make the constant potentially very long.  It was really just a conscious style choice, but if you feel really strongly, I could be swayed.

Reply:
I can't say I feel "really strongly," but I guess it's a different conscious style choice-- I tend to like things more compact.  It will only remain consistent if everybody knows/remembers to break up these calls the same way.  But I guess we're not likely to make many more/new tests at this point, so that doesn't really matter...?